style.css

```diff
@@ -1,652 +1,758 @@
-@font-face {
-  font-family: Fraunces-font;
-  src: url(./assets/Fraunces-VariableFont_SOFT\,WONK\,opsz\,wght.ttf);
-}
-
-@font-face {
-  font-family: Fraunces-font-bold;
-  src: url(./assets/static/Fraunces_72pt\,Soft-Bold.ttf);
-}
-
-footer {
-  color: gray;
-}
-
-.date {
-  color: white;
-}
-
-.container {
-  width: 100%;
-}
-
-.one {
-  background-color: #f72210;
-}
-
-.btn-warning {
-  font-size: 6px;
-  border: 1px solid rgba(240, 173, 78, 0.75);
-  border-radius: 40px;
-  background: yellow;
-  color: black;
-  font-weight: bold;
-}
-
-.nav-login-text {
-  color: white;
-  font-size: 1.3rem;
-}
-
-.nav-subscribe-button {
-  font-size: 0.8rem;
-}
-
-.two {
-  font-weight: bold;
-}
-
-.gray {
-  color: gray;
-  font-size: 11px;
-}
-
-.images-float-left {
-  float: left;
-}
-
-.images-float-right {
-  float: right;
-}
-
-.search {
-  font-weight: lighter;
-}
-
-.news {
-  color: white;
-  font-size: 1.8rem;
-  font-weight: bolder;
-  font-family: Fraunces-font-bold, 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-}
-
-.news_red {
-  fill: #f72210;
-}
-
-.big {
-  font-size: 2.5rem;
-  font-family: Fraunces-font-bold, 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-  font-weight: bolder;
-}
-
-.three {
-  font-size: 0.9rem;
-  font-style: bolder;
-  color: gray;
-}
-
-.footer-copyright {
-  font-size: 0.8rem;
-}
-
-.edition {
-  font-weight: bolder;
-}
-
-.polaska,
-.pakistan,
-.china,
-.romania,
-.usa,
-.edition {
-  font-size: 0.8rem;
-}
-
-.four {
-  display: flex;
-  align-items: center;
-}

```

Comment:
Use more of bootstrap styling than custom styling. Example in the above, you can use `d-flex` and `align-items-center` bootstrap classes. Make these change for all your css file. Only use custom styling where bootstrap styling is not possible.